Documentation/ApiOverview/Localization/LocalizationApi/LanguageService.rst

```diff
@@ -12,5 +12,37 @@ be created directly, therefore its constructor is internal. Create a
 
 In the backend context a :php:`LanguageService` is stored in the global
 variable :php:`$GLOBALS['LANG']`.
+In the frontend a :php:`LanguageService` can be accessed via the contentObject:
+
+..  code-block:: php
+    :caption: Classes/Controller/ExampleController.php
+
+    use Psr\Http\Message\ServerRequestInterface;
+    use TYPO3\CMS\Core\Localization\LanguageServiceFactory;
+    use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+    class ExampleController {
+        proteced ServerRequestInterface $request;
+
+        public function processAction(ServerRequestInterface $request): string
```

Comment:
Please enlighten me: how is processAction() called in frontend and passing the request object?

Reply:
See original patch from Franz:
`:caption: NOT Extbase example Classes/Controller/ExampleController.php`

Reply:
So a USER/USER_INT? But then you have three arguments passed to the method. Sorry, I don't get it. How is the action then called?